tests/mdal_testutils.cpp

```diff
@@ -383,14 +384,16 @@ void saveAndCompareMesh( const std::string &filename, const std::string &savedFi
   if ( !meshName.empty() )
     uri = "\"" + uri + "\":" + meshName;
 
+  std::string savedUri = MDAL::buildMeshUri( savedFile, meshName, driver );
+
   // Open mesh
   MDAL_MeshH meshToSave = MDAL_LoadMesh( uri.c_str() );
   EXPECT_NE( meshToSave, nullptr );
   MDAL_Status s = MDAL_LastStatus();
   ASSERT_EQ( MDAL_Status::None, s );
 
   // Save the mesh
-  MDAL_SaveMesh( meshToSave, savedFile.c_str(), driver.c_str() );
```

Comment:
can we keep boths MDAL_SaveMesh and MDAL_SaveMeshWithURi in the test to test both of them (both are still in public API)

Reply:
done